news/2022-12-24-results-osu!taiko-featured-artist-cup-3.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "Results - osu!taiko Featured Artist Cup #3"
+date: 2022-11-24 10:00:00 +0000
+---
+
+The battle is over! Come join us in unveiling the winners of the osu!taiko mapping showdown of the year!
+
+![](https://assets.ppy.sh/contests/155/header.jpg)
+
+2 months ago, the [osu!taiko Featured Artist Cup](/wiki/Contests/o!tFAC) opened its gates to the world for the [third time](https://osu.ppy.sh/home/news/2022-10-23-osutaiko-featured-artist-cup-3), and after a careful process of judging, we're finally ready to crown the winners of this iteration!
+
+Before we proceed, we'd like to announce that because of the very high amount of entries, and the sheer quality of the submissions, we will be awarding the top 3 teams with badges, instead of only the winning team! Check out the [following page](/wiki/Contests/Monthly_Beatmapping_Contest#rewards) for a detailed list of the rewards.
```

Comment:
``` Suggestion
Before we proceed, we'd like to announce that due to the high amount of entries, and the sheer quality of the submissions, we will be awarding the **top 3 teams** with badges, instead of only the winning team! Check out the [following page](/wiki/Contests/Monthly_Beatmapping_Contest#rewards) for a detailed list of the rewards.
```